remote-access/ssh/passwordless.md

```diff
@@ -83,9 +84,9 @@ If this did not work, delete your keys with `rm ~/.ssh/id*` and follow the instr
 
 You can also send files over SSH using the `scp` command (secure copy). See the [SCP guide](scp.md) for more information.
 
-## Let macOS store your passphrase so you don't have to enter it each time
+## Store the passphrase in the macOS Keychain
 
-If you're using macOS and after verifying that your new key allows you to connect, you can optionally choose to store the passphrase for your key in the macOS Keychain. This will make it so that you don't have to enter the passphrase each time you connect to your Pi.
+If you are using macOS, and after verifying that your new key allows you to connect, you can optionally choose to store the passphrase for your key in the macOS Keychain. This allows you to connect to your Raspberry Pi without entering the passphrase.
```

Comment:
"the passphrase for your key" -> "the passphrase for your key (if you've set one)" ?

Reply:
It should be obvious from the section title that this only applies in the case where the user has set a passphrase. You can't ask macOS to store a passphrase if there isn't one.

Reply:
Meh. It may be obvious to experienced users like yourself, but IMVHO it's always better to be explicit to avoid doubt / prevent confusion :slightly_smiling_face: